sound/soc/intel/common/soc-acpi-intel-cnl-match.c

```diff
@@ -37,6 +44,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cnl_machines[] = {
 		.sof_fw_filename = "sof-cnl.ri",
 		.sof_tplg_filename = "sof-cml-rt5682.tplg",
 	},
+	{
+		.id = "MX98357A",
+		.drv_name = CML_MACHINE_DRIVER,
```

Comment:
Better to just use the driver name directly for readability, the define doesn't add much value here.

Reply:
@plbossart fixed now